paws/templates/public.yaml

```diff
@@ -120,6 +120,9 @@ spec:
     http:
       paths:
       - backend:
-          serviceName: paws-public
-          servicePort: 8000
+          service:
+            name: paws-public
+            port:
+             number: 8000
+        pathType: ImplementationSpecific
```

Comment:
Why is this using `ImplementationSpecific` and not `Prefix`? Also looks like the `path` attribute is missing

Reply:
Mostly the lack of path attribute in the existing config led me to leave it as the default `ImplementationSpecific`. As trying to set it as Prefix would require a path, but I wasn't sure what that path would be.

Reply:
I'm guessing we could use just `/` to match everything.

Reply:
Loads in minikube. Seems good!